Index: vendor/shopware/core/Framework/Adapter/Twig/NamespaceHierarchy/BundleHierarchyBuilder.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/vendor/shopware/core/Framework/Adapter/Twig/NamespaceHierarchy/BundleHierarchyBuilder.php b/vendor/shopware/core/Framework/Adapter/Twig/NamespaceHierarchy/BundleHierarchyBuilder.php
--- a/vendor/shopware/core/Framework/Adapter/Twig/NamespaceHierarchy/BundleHierarchyBuilder.php
+++ b/vendor/shopware/core/Framework/Adapter/Twig/NamespaceHierarchy/BundleHierarchyBuilder.php	(date 1746357602993)
@@ -63,6 +63,7 @@
      */
     private function getAppTemplateNamespaces(): array
     {
+        return []; // This is an unsafe call to the database.
         return $this->connection->fetchAllAssociativeIndexed(
             'SELECT `app`.`name`, `app`.`version`, `app`.`template_load_priority`
              FROM `app`
